cyclestatus/cycle_status.py

```diff
@@ -37,10 +37,14 @@ def positive_int(arg: str) -> int:
     return ret
 
 
+_bot_guild_var = r"{bot_guild_len}"
```

Comment:
Speaking of naming conventions, it's probably best to use ``count`` instead of ``len``.